### ISSUE TITLE
Album cover positioning and scale isn't updated to adapt to new terminal size

### ISSUE THREAD

**Issue by Porkepix** [tramhao/termusic#250]
When changing the size of the terminal (many situations where this can happen), the general TUI is resized and adapted to the new terminal size.
The album cover however gets stuck to the size it knew when Termusic started, even when moving to a new track. Going back to the initial size is fine, but any other size will result to a cover of the wrong size (going out of the terminal because too big if we reduce its size, or being ridiculously small if we made the terminal much bigger), and not located the way it should be.

**Comment by tramhao** [comment]
Yes, it's a headache now. I want to switch to ratatui-image. It's better to integrate into user interface. Currently it's hovering.

**Comment by tramhao** [comment]
By the way, for the time being you can manually adjust the size of cover on the fly.
